shared/src/data/GoodNumber.ts

```diff
@@ -211,6 +212,10 @@ export class GN {
     return this.mul(this);
   }
 
+  reciprocal() {
+    return new GN(this.scaler, this.resolution, this.base).div(this);
+  }
```

Comment:
This won't work in most cases. Any time the number is greater than its `scaler`, this will return 0. Moreover, seeing as other big number libraries (JSBI, Big, BigNumber) don't have a `reciprocal` function, I'm not sure we should either.